lib/WebService/Async/SmartyStreets.pm

```diff
@@ -0,0 +1,177 @@
+package WebService::Async::SmartyStreets;
+# ABSTRACT: Access SmartyStreet API
+
+use strict;
+use warnings;
+
+our $VERSION = '0.001';
+
+=HEAD
+
+WebService::Async::SmartyStreets - calls the SmartyStreets API and checks for the validity of the address
+
+=head1 VERSION
+
+version 0.001
+
+=head1 SYNOPSIS
+    
+    my $loop = IO::Async::Loop->new;
+    $loop->add(
+        my $ss = WebService::Async::SmartyStreets->new(
+            # International token
+            auth_id => '...'
+            token => '...'
+        )
+    );
+    (async sub {
+        my $addr = await $ss->verify_international(
+            # insert address here
+        );
+    })->()->get;
+    
+=head1 DESCRIPTION
+
+his class calls the SmartyStreets API and parse the response to `WebService::Async::SmartyStreets::Address`
```

Comment:
```suggestion
This class calls the SmartyStreets API and parse the response to `WebService::Async::SmartyStreets::Address`
```

Also probably good to note the usage of Future::AsyncAwait here, alongside references to the public docs of SmartyStreets API.